packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -157,7 +157,10 @@ export function DatePicker({
                     data-testid="jkl-datepicker__input"
                     value={state.dateString}
                     onFocus={handleFocusChange(onFocus)}
-                    onBlur={handleFocusChange(onBlur)}
+                    onBlur={(e) => {
```

Comment:
Kan du dra dette ut som en funksjon please.

Reply:
> Vil gjerne ha en commit på dette :)

Working on it! Takk for gode tilbakemeldinger 👍🏻